src/test/java/edu/kis/vh/nursery/FIFORyhmerJUnitTest.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.vh.nursery;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+
+public class FIFORyhmerJUnitTest {
+	FIFORyhmer ryhmer = new FIFORyhmer();
+	final int IS_EMPTY_INDICATOR = -1;
```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).